### PR TITLE
pow pow pow

### PR DESCRIPTION
**Pull request by dfarr**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/foo/pull/14" target="_blank">ReviewNinja

**Comment by review-ninja** [comment]
<a href="https://staging.review.ninja/reviewninja/foo/pull/14" target="_blank">ReviewNinja

File: math.js [hunk 1]
```diff
@@ -14,5 +14,9 @@ module.exports = {
   
   multiply: function(a, b) {
     return a * b;
+  },
+  
+  pow: function(a, b) {
+    return a ^ b;
```
Comment (dfarr) [inline math.js]: huh !fix

**Comment by dfarr** [comment]
👍 

👎 

⭐

**Comment by dfarr** [comment]
👍

**Comment by dfarr** [comment]
👎

**Comment by dfarr** [comment]
⭐

**Comment by dfarr** [comment]
⭐

File: math.js (hunk 1 above)
Comment (casche) [inline math.js]: !resolved

Comment (casche) [inline math.js]: !fix it's not fixed yet

Comment (casche) [inline math.js]: !resolved

File: math.js [hunk 2]
```diff
@@ -14,5 +14,9 @@ module.exports = {
   
   multiply: function(a, b) {
     return a * b;
+  },
```
Comment (casche) [inline math.js]: !fix this line has too much code

File: math.js [hunk 3]
```diff
@@ -14,5 +14,9 @@ module.exports = {
...
+
```
Comment (casche) [inline math.js]: !fix this problem

Comment (casche) [inline math.js]: !fixed

File: math.js [hunk 4]
```diff
@@ -14,5 +14,9 @@ module.exports = {
   
   multiply: function(a, b) {
     return a * b;
+  },
+  
+  pow: function(a, b) {
```
Comment (casche) [inline math.js]: !fix this line of code David!

Comment (casche) [inline math.js]: !fixed

File: math.js (hunk 2 above)
Comment (casche) [inline math.js]: !resolved

Comment (casche) [inline math.js]: !fix this issue

Comment (casche) [inline math.js]: !fixed

**Pull request merged**